scripts/convert-databases.bash

```diff
@@ -0,0 +1,225 @@
+	l2chaindata_status="converted"
+else
+	if $skip_existing; then
+		echo "== l2chaindata directory already exists, skipping conversion (--skip-existing flag is set)"
+		l2chaindata_status="skipped"
+	else
+		# unreachable, we already had to remove root directory
+		echo script error, reached unreachable
+		exit 1
+	fi
+fi
+
+if ! [ -e $dst/l2chaindata/ancient ]; then
+	echo "== Copying l2chaindata ancients"
+	(set -x; cp -r $src/l2chaindata/ancient $dst/l2chaindata/) || (l2chaindata_ancient_status="failed to copy"; printStatus; exit 1)
+	l2chaindata_ancient_status="copied"
+else
+	if $skip_existing; then
+		echo "== l2chaindata/ancient directory already exists, skipping copy (--skip-existing flag is set)"
+		l2chaindata_ancient_status="skipped"
+	else
+		# unreachable, we already had to remove root directory
+		echo script error, reached unreachable
+		exit 1
+	fi
+fi
+
+if ! [ -e $dst/arbitrumdata ]; then
+	echo "== Converting arbitrumdata db"
+	(set -x; $dbconv --src.db-engine="leveldb" --src.data $src/arbitrumdata --dst.db-engine="pebble" --dst.data $dst/arbitrumdata --convert --compact) || (arbitrumdata_status="conversion failed"; printStatus; exit 1)
+	arbitrumdata_status="converted"
+else
+	if $skip_existing; then
+		echo "== arbitrumdata directory already exists, skipping conversion (--skip-existing flag is set)"
+		arbitrumdata_status="skipped"
+	else
+		# unreachable, we already had to remove root directory
+		echo script error, reached unreachable
+		exit 1
+	fi
+fi
+
+if [ -e $src/wasm ]; then
+	if ! [ -e $dst/wasm ]; then
+		echo "== Converting wasm db"
+		(set -x; $dbconv --src.db-engine="leveldb" --src.data $src/wasm --dst.db-engine="pebble" --dst.data $dst/wasm --convert --compact) || (wasm_status="conversion failed"; printStatus; exit 1)
+		wasm_status="converted"
+	else
+		if $skip_existing; then
+			echo "== wasm directory already exists, skipping conversion (--skip-existing flag is set)"
+			wasm_status="skipped"
+		else
+			# unreachable, we already had to remove root directory
+			echo script error, reached unreachable
+			exit 1
+		fi
+	fi
+else
+	echo "== Note: Source directory does not contain wasm database."
+	wasm_status="not found in source directory"
+fi
+
+if [ -e $src/classic-msg ]; then
+	if ! [ -e $dst/classic-msg ]; then
+		echo "== Converting classic-msg db"
+		(set -x; $dbconv --src.db-engine="leveldb" --src.data $src/classic-msg --dst.db-engine="pebble" --dst.data $dst/classic-msg --convert --compact) || (classicmsg_status="conversion failed"; printStatus; exit 1)
+		classicmsg_status="converted"
+	else
+		if $skip_existing; then
+			echo "== classic-msg directory already exists, skipping conversion (--skip-existing flag is set)"
+			classicmsg_status="skipped"
+		else
+			# unreachable, we already had to remove root directory
+			echo script error, reached unreachable
+			exit 1
+		fi
+	fi
```

Comment:
nitpick: this same logic is implemented several, one for each converted database.
This indicates that this logic would be better placed in a function :)